integration_test/Bindings/Python/dialects/comb.py

```diff
@@ -0,0 +1,158 @@
+# REQUIRES: bindings_python
+# RUN: %PYTHON% %s | FileCheck %s
+
+import circt
+from circt.design_entry import connect
+from circt.dialects import comb, hw
+
+from mlir.ir import Context, Location, InsertionPoint, IntegerType, IntegerAttr, Module
+
+with Context() as ctx, Location.unknown():
+  circt.register_dialects(ctx)
+
+  i32 = IntegerType.get_signless(32)
+
+  m = Module.create()
+  with InsertionPoint(m.body):
+
+    def build(module):
+      # CHECK: %[[CONST:.+]] = hw.constant 1 : i32
+      const = hw.ConstantOp(i32, IntegerAttr.get(i32, 1))
+
+      # CHECK: comb.extract %[[CONST]] from 14
+      comb.ExtractOp.create(14, i32, {"input": const.result})
+      # CHECK: comb.extract %[[CONST]] from 14
+      extract = comb.ExtractOp.create(14, i32)
+      connect(extract.input, const.result)
+
+      # CHECK: comb.parity %[[CONST]]
+      comb.ParityOp.create(i32, {"input": const.result})
+      # CHECK: comb.parity %[[CONST]]
+      parity = comb.ParityOp.create(i32)
+      connect(parity.input, const.result)
+
+      # CHECK: comb.sext %[[CONST]]
+      comb.SExtOp.create(i32, {"input": const.result})
+      # CHECK: comb.sext %[[CONST]]
+      sext = comb.SExtOp.create(i32)
+      connect(sext.input, const.result)
+
+      # CHECK: comb.divs %[[CONST]], %[[CONST]]
+      comb.DivSOp.create(i32, {"lhs": const.result, "rhs": const.result})
```

Comment:
Yeah, this syntax makes my eyes bleed as well. I see where you're coming from -- kwargs is the right way to go.

Reply:
Yep, will be addressed next.

Reply:
Also, if the kwargs are two verbose for these unary and binary ops, I'm thinking these could be eventually updated to use whatever form ends up being best for the variadic ops.